assets/js/vue/components/search/ResultSet.vue

```diff
@@ -3,7 +3,7 @@
         <section class="section-content pt-3">
             <div class="row d-flex flex-row justify-content-center">
                 <h5>
-                    Found {{ results.count }} results for: "{{ results.formValues.query }}"
+                    Found #{{ results.count }} results"
```

Comment:
Last double quote is not needed.

```suggestion
                    Found #{{ results.count }} results
```